SdkVersion.props

```diff
@@ -6,16 +6,17 @@
 	<SDKVersionRS3>10.0.16299.0</SDKVersionRS3>
 	<SDKVersionRS4>10.0.17134.0</SDKVersionRS4>
 	<SDKVersionRS5>10.0.17763.0</SDKVersionRS5>
-	<SDKVersion19H1Insider>10.0.18323.0</SDKVersion19H1Insider>
+	<SDKVersion19H1>10.0.18362.0</SDKVersion19H1>
+	<SDKVersionInsider>10.0.18362.0</SDKVersionInsider>
   </PropertyGroup>
   <PropertyGroup>
     <UseInsiderSDK>true</UseInsiderSDK>
```

Comment:
Should you remove this line? Here we set UseInsiderSDK to true. and we don't need to use conditional in MuxSdkVersion

Reply:
We currently aren't using an insider SDK but i'm sure we will again soon.